citesphere/pom.xml

```diff
@@ -23,7 +23,7 @@
 
         <javers.version>5.0.0</javers.version>
         <spring.kafka.version>2.2.6.RELEASE</spring.kafka.version>
-        <spring-social-zotero.version>0.10</spring-social-zotero.version>
+        <spring-social-zotero.version>0.11</spring-social-zotero.version>
```

Comment:
I don't think this release exists?

Reply:
Not yet. But there is a corresponding plugin PR.

Reply:
I think I made comment to move the changes from the plugin into citesphere directly?

Reply:
Never mind, I think I confused that with the other PR.